crates/symbolicator-service/src/caching/metadata.rs

```diff
@@ -0,0 +1,69 @@
+use std::time::SystemTime;
+
+use serde::{Deserialize, Serialize};
+
+use crate::types::Scope;
+
+use super::CacheEntry;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Metadata {
+    pub scope: Scope,
+    pub time_created: SystemTime,
+}
+
+/// A cache entry with optional metadata.
+#[derive(Debug, Clone)]
+pub struct MdCacheEntry<T> {
```

Comment:
nit: `Md` might confuse readers, should we call this `CacheEntry` and the original `CacheEntry` `CacheContents` or similar?

Reply:
I had the exact same thought, `CacheEntry` -> `CacheContents` and `MdCacheEntry` -> `CacheEntry`.